diff --git a/mediapipe/calculators/audio/mfcc_mel_calculators.proto b/mediapipe/calculators/audio/mfcc_mel_calculators.proto
index 89af5eb4..beefba53 100644
--- a/mediapipe/calculators/audio/mfcc_mel_calculators.proto
+++ b/mediapipe/calculators/audio/mfcc_mel_calculators.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/audio/rational_factor_resample_calculator.proto b/mediapipe/calculators/audio/rational_factor_resample_calculator.proto
index 97d7f202..8d79ebc6 100644
--- a/mediapipe/calculators/audio/rational_factor_resample_calculator.proto
+++ b/mediapipe/calculators/audio/rational_factor_resample_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/audio/spectrogram_calculator.proto b/mediapipe/calculators/audio/spectrogram_calculator.proto
index ddfca1d1..a111f761 100644
--- a/mediapipe/calculators/audio/spectrogram_calculator.proto
+++ b/mediapipe/calculators/audio/spectrogram_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/audio/stabilized_log_calculator.proto b/mediapipe/calculators/audio/stabilized_log_calculator.proto
index ef6faa8e..7ab694af 100644
--- a/mediapipe/calculators/audio/stabilized_log_calculator.proto
+++ b/mediapipe/calculators/audio/stabilized_log_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/audio/time_series_framer_calculator.proto b/mediapipe/calculators/audio/time_series_framer_calculator.proto
index 9e5b0746..4fa4c1eb 100644
--- a/mediapipe/calculators/audio/time_series_framer_calculator.proto
+++ b/mediapipe/calculators/audio/time_series_framer_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/bypass_calculator.proto b/mediapipe/calculators/core/bypass_calculator.proto
index 1a273edb..cf06ebf4 100644
--- a/mediapipe/calculators/core/bypass_calculator.proto
+++ b/mediapipe/calculators/core/bypass_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/concatenate_vector_calculator.proto b/mediapipe/calculators/core/concatenate_vector_calculator.proto
index bddb8af9..ee7bc8d5 100644
--- a/mediapipe/calculators/core/concatenate_vector_calculator.proto
+++ b/mediapipe/calculators/core/concatenate_vector_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/constant_side_packet_calculator.proto b/mediapipe/calculators/core/constant_side_packet_calculator.proto
index 78a773a6..a04510eb 100644
--- a/mediapipe/calculators/core/constant_side_packet_calculator.proto
+++ b/mediapipe/calculators/core/constant_side_packet_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/framework/formats/classification.proto";
diff --git a/mediapipe/calculators/core/dequantize_byte_array_calculator.proto b/mediapipe/calculators/core/dequantize_byte_array_calculator.proto
index 3032dbf4..7c2e852d 100644
--- a/mediapipe/calculators/core/dequantize_byte_array_calculator.proto
+++ b/mediapipe/calculators/core/dequantize_byte_array_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/flow_limiter_calculator.proto b/mediapipe/calculators/core/flow_limiter_calculator.proto
index e969abf3..80ee7723 100644
--- a/mediapipe/calculators/core/flow_limiter_calculator.proto
+++ b/mediapipe/calculators/core/flow_limiter_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/gate_calculator.proto b/mediapipe/calculators/core/gate_calculator.proto
index b7d597a6..142b6951 100644
--- a/mediapipe/calculators/core/gate_calculator.proto
+++ b/mediapipe/calculators/core/gate_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/get_vector_item_calculator.proto b/mediapipe/calculators/core/get_vector_item_calculator.proto
index 9cfb579e..585aec41 100644
--- a/mediapipe/calculators/core/get_vector_item_calculator.proto
+++ b/mediapipe/calculators/core/get_vector_item_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/graph_profile_calculator.proto b/mediapipe/calculators/core/graph_profile_calculator.proto
index 88b405e5..a8bfda94 100644
--- a/mediapipe/calculators/core/graph_profile_calculator.proto
+++ b/mediapipe/calculators/core/graph_profile_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/packet_cloner_calculator.proto b/mediapipe/calculators/core/packet_cloner_calculator.proto
index 29768d1e..50f1f2ac 100644
--- a/mediapipe/calculators/core/packet_cloner_calculator.proto
+++ b/mediapipe/calculators/core/packet_cloner_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/packet_resampler_calculator.proto b/mediapipe/calculators/core/packet_resampler_calculator.proto
index 29ca8082..73ec2de3 100644
--- a/mediapipe/calculators/core/packet_resampler_calculator.proto
+++ b/mediapipe/calculators/core/packet_resampler_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/packet_thinner_calculator.proto b/mediapipe/calculators/core/packet_thinner_calculator.proto
index 34fd9bc3..18ae3887 100644
--- a/mediapipe/calculators/core/packet_thinner_calculator.proto
+++ b/mediapipe/calculators/core/packet_thinner_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/quantize_float_vector_calculator.proto b/mediapipe/calculators/core/quantize_float_vector_calculator.proto
index 3f6cfda2..98a12890 100644
--- a/mediapipe/calculators/core/quantize_float_vector_calculator.proto
+++ b/mediapipe/calculators/core/quantize_float_vector_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/sequence_shift_calculator.proto b/mediapipe/calculators/core/sequence_shift_calculator.proto
index 36b0bb95..c2bffda6 100644
--- a/mediapipe/calculators/core/sequence_shift_calculator.proto
+++ b/mediapipe/calculators/core/sequence_shift_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/core/split_vector_calculator.proto b/mediapipe/calculators/core/split_vector_calculator.proto
index 53acbb7b..04b00193 100644
--- a/mediapipe/calculators/core/split_vector_calculator.proto
+++ b/mediapipe/calculators/core/split_vector_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/bilateral_filter_calculator.proto b/mediapipe/calculators/image/bilateral_filter_calculator.proto
index a787437d..5ee758cc 100644
--- a/mediapipe/calculators/image/bilateral_filter_calculator.proto
+++ b/mediapipe/calculators/image/bilateral_filter_calculator.proto
@@ -2,6 +2,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/image_cropping_calculator.proto b/mediapipe/calculators/image/image_cropping_calculator.proto
index 55d3467d..45196436 100644
--- a/mediapipe/calculators/image/image_cropping_calculator.proto
+++ b/mediapipe/calculators/image/image_cropping_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/image_transformation_calculator.proto b/mediapipe/calculators/image/image_transformation_calculator.proto
index 739c5bfb..1da16915 100644
--- a/mediapipe/calculators/image/image_transformation_calculator.proto
+++ b/mediapipe/calculators/image/image_transformation_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/calculators/image/rotation_mode.proto";
 import "mediapipe/framework/calculator.proto";
diff --git a/mediapipe/calculators/image/mask_overlay_calculator.proto b/mediapipe/calculators/image/mask_overlay_calculator.proto
index 3b82d610..8ff7f3a2 100644
--- a/mediapipe/calculators/image/mask_overlay_calculator.proto
+++ b/mediapipe/calculators/image/mask_overlay_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/opencv_encoded_image_to_image_frame_calculator.proto b/mediapipe/calculators/image/opencv_encoded_image_to_image_frame_calculator.proto
index 3ec29bdd..9cf22b65 100644
--- a/mediapipe/calculators/image/opencv_encoded_image_to_image_frame_calculator.proto
+++ b/mediapipe/calculators/image/opencv_encoded_image_to_image_frame_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/opencv_image_encoder_calculator.proto b/mediapipe/calculators/image/opencv_image_encoder_calculator.proto
index 0564fb27..16120ae8 100644
--- a/mediapipe/calculators/image/opencv_image_encoder_calculator.proto
+++ b/mediapipe/calculators/image/opencv_image_encoder_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/recolor_calculator.proto b/mediapipe/calculators/image/recolor_calculator.proto
index abbf0849..584c9a3c 100644
--- a/mediapipe/calculators/image/recolor_calculator.proto
+++ b/mediapipe/calculators/image/recolor_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/image/rotation_mode.proto b/mediapipe/calculators/image/rotation_mode.proto
index 7fa4a8ed..08d8c1d4 100644
--- a/mediapipe/calculators/image/rotation_mode.proto
+++ b/mediapipe/calculators/image/rotation_mode.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 option java_package = "com.google.mediapipe.calculator.proto";
 option java_outer_classname = "RotationModeProto";
diff --git a/mediapipe/calculators/image/scale_image_calculator.proto b/mediapipe/calculators/image/scale_image_calculator.proto
index 2b7572d5..e6f8c2a8 100644
--- a/mediapipe/calculators/image/scale_image_calculator.proto
+++ b/mediapipe/calculators/image/scale_image_calculator.proto
@@ -2,6 +2,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/framework/formats/image_format.proto";
diff --git a/mediapipe/calculators/image/segmentation_smoothing_calculator.proto b/mediapipe/calculators/image/segmentation_smoothing_calculator.proto
index 12b10ccd..fc761aef 100644
--- a/mediapipe/calculators/image/segmentation_smoothing_calculator.proto
+++ b/mediapipe/calculators/image/segmentation_smoothing_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/set_alpha_calculator.proto b/mediapipe/calculators/image/set_alpha_calculator.proto
index 3f07ddb3..4cdbd84f 100644
--- a/mediapipe/calculators/image/set_alpha_calculator.proto
+++ b/mediapipe/calculators/image/set_alpha_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/image/warp_affine_calculator.proto b/mediapipe/calculators/image/warp_affine_calculator.proto
index 20e6c1b0..b87e1fc6 100644
--- a/mediapipe/calculators/image/warp_affine_calculator.proto
+++ b/mediapipe/calculators/image/warp_affine_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/gpu/gpu_origin.proto";
diff --git a/mediapipe/calculators/internal/callback_packet_calculator.proto b/mediapipe/calculators/internal/callback_packet_calculator.proto
index 6a5cfb05..b5444e26 100644
--- a/mediapipe/calculators/internal/callback_packet_calculator.proto
+++ b/mediapipe/calculators/internal/callback_packet_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/audio_to_tensor_calculator.proto b/mediapipe/calculators/tensor/audio_to_tensor_calculator.proto
index 5b7d61bc..a16f8544 100644
--- a/mediapipe/calculators/tensor/audio_to_tensor_calculator.proto
+++ b/mediapipe/calculators/tensor/audio_to_tensor_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/bert_preprocessor_calculator.proto b/mediapipe/calculators/tensor/bert_preprocessor_calculator.proto
index 72b56914..bfc4091c 100644
--- a/mediapipe/calculators/tensor/bert_preprocessor_calculator.proto
+++ b/mediapipe/calculators/tensor/bert_preprocessor_calculator.proto
@@ -16,6 +16,7 @@ limitations under the License.
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/feedback_tensors_calculator.proto b/mediapipe/calculators/tensor/feedback_tensors_calculator.proto
index ac36b678..cc44ed58 100644
--- a/mediapipe/calculators/tensor/feedback_tensors_calculator.proto
+++ b/mediapipe/calculators/tensor/feedback_tensors_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/image_to_tensor_calculator.proto b/mediapipe/calculators/tensor/image_to_tensor_calculator.proto
index 780ee802..9683a0cf 100644
--- a/mediapipe/calculators/tensor/image_to_tensor_calculator.proto
+++ b/mediapipe/calculators/tensor/image_to_tensor_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/gpu/gpu_origin.proto";
diff --git a/mediapipe/calculators/tensor/inference_calculator.proto b/mediapipe/calculators/tensor/inference_calculator.proto
index 78a0039b..72fc7813 100644
--- a/mediapipe/calculators/tensor/inference_calculator.proto
+++ b/mediapipe/calculators/tensor/inference_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/framework/calculator_options.proto";
diff --git a/mediapipe/calculators/tensor/landmarks_to_tensor_calculator.proto b/mediapipe/calculators/tensor/landmarks_to_tensor_calculator.proto
index 6ef1c8d4..6704491e 100644
--- a/mediapipe/calculators/tensor/landmarks_to_tensor_calculator.proto
+++ b/mediapipe/calculators/tensor/landmarks_to_tensor_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/regex_preprocessor_calculator.proto b/mediapipe/calculators/tensor/regex_preprocessor_calculator.proto
index 793067a8..d161f6ff 100644
--- a/mediapipe/calculators/tensor/regex_preprocessor_calculator.proto
+++ b/mediapipe/calculators/tensor/regex_preprocessor_calculator.proto
@@ -16,6 +16,7 @@ limitations under the License.
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensor_converter_calculator.proto b/mediapipe/calculators/tensor/tensor_converter_calculator.proto
index 97c2154a..59934b00 100644
--- a/mediapipe/calculators/tensor/tensor_converter_calculator.proto
+++ b/mediapipe/calculators/tensor/tensor_converter_calculator.proto
@@ -1,6 +1,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensors_to_audio_calculator.proto b/mediapipe/calculators/tensor/tensors_to_audio_calculator.proto
index 90762712..b36226fc 100644
--- a/mediapipe/calculators/tensor/tensors_to_audio_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_audio_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensors_to_classification_calculator.proto b/mediapipe/calculators/tensor/tensors_to_classification_calculator.proto
index 32bc4b63..f4346fcc 100644
--- a/mediapipe/calculators/tensor/tensors_to_classification_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_classification_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/label_map.proto";
diff --git a/mediapipe/calculators/tensor/tensors_to_detections_calculator.proto b/mediapipe/calculators/tensor/tensors_to_detections_calculator.proto
index c9d6b69d..2f7ff05e 100644
--- a/mediapipe/calculators/tensor/tensors_to_detections_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_detections_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensors_to_floats_calculator.proto b/mediapipe/calculators/tensor/tensors_to_floats_calculator.proto
index 69405019..4ba2ee96 100644
--- a/mediapipe/calculators/tensor/tensors_to_floats_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_floats_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensors_to_landmarks_calculator.proto b/mediapipe/calculators/tensor/tensors_to_landmarks_calculator.proto
index 2608a145..5f394673 100644
--- a/mediapipe/calculators/tensor/tensors_to_landmarks_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_landmarks_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensor/tensors_to_segmentation_calculator.proto b/mediapipe/calculators/tensor/tensors_to_segmentation_calculator.proto
index 1662576b..574745ca 100644
--- a/mediapipe/calculators/tensor/tensors_to_segmentation_calculator.proto
+++ b/mediapipe/calculators/tensor/tensors_to_segmentation_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/gpu/gpu_origin.proto";
diff --git a/mediapipe/calculators/tensorflow/graph_tensors_packet_generator.proto b/mediapipe/calculators/tensorflow/graph_tensors_packet_generator.proto
index 1196ca1e..9bbe8d6d 100644
--- a/mediapipe/calculators/tensorflow/graph_tensors_packet_generator.proto
+++ b/mediapipe/calculators/tensorflow/graph_tensors_packet_generator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/packet_generator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/image_frame_to_tensor_calculator.proto b/mediapipe/calculators/tensorflow/image_frame_to_tensor_calculator.proto
index c48e6a86..5114c111 100644
--- a/mediapipe/calculators/tensorflow/image_frame_to_tensor_calculator.proto
+++ b/mediapipe/calculators/tensorflow/image_frame_to_tensor_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "tensorflow/core/framework/types.proto";
diff --git a/mediapipe/calculators/tensorflow/lapped_tensor_buffer_calculator.proto b/mediapipe/calculators/tensorflow/lapped_tensor_buffer_calculator.proto
index bcd14985..14f45dfb 100644
--- a/mediapipe/calculators/tensorflow/lapped_tensor_buffer_calculator.proto
+++ b/mediapipe/calculators/tensorflow/lapped_tensor_buffer_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/matrix_to_tensor_calculator_options.proto b/mediapipe/calculators/tensorflow/matrix_to_tensor_calculator_options.proto
index 50a1775c..aa722760 100644
--- a/mediapipe/calculators/tensorflow/matrix_to_tensor_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/matrix_to_tensor_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/object_detection_tensors_to_detections_calculator.proto b/mediapipe/calculators/tensorflow/object_detection_tensors_to_detections_calculator.proto
index 14914468..dd152af5 100644
--- a/mediapipe/calculators/tensorflow/object_detection_tensors_to_detections_calculator.proto
+++ b/mediapipe/calculators/tensorflow/object_detection_tensors_to_detections_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/pack_media_sequence_calculator.proto b/mediapipe/calculators/tensorflow/pack_media_sequence_calculator.proto
index 6ba09fb1..f21c0f37 100644
--- a/mediapipe/calculators/tensorflow/pack_media_sequence_calculator.proto
+++ b/mediapipe/calculators/tensorflow/pack_media_sequence_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "tensorflow/core/example/feature.proto";
diff --git a/mediapipe/calculators/tensorflow/tensor_squeeze_dimensions_calculator.proto b/mediapipe/calculators/tensorflow/tensor_squeeze_dimensions_calculator.proto
index 87c59780..ac6e3cea 100644
--- a/mediapipe/calculators/tensorflow/tensor_squeeze_dimensions_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensor_squeeze_dimensions_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensor_to_image_frame_calculator.proto b/mediapipe/calculators/tensorflow/tensor_to_image_frame_calculator.proto
index 3410068d..8e046e35 100644
--- a/mediapipe/calculators/tensorflow/tensor_to_image_frame_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensor_to_image_frame_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensor_to_matrix_calculator.proto b/mediapipe/calculators/tensorflow/tensor_to_matrix_calculator.proto
index e8647700..d494f010 100644
--- a/mediapipe/calculators/tensorflow/tensor_to_matrix_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensor_to_matrix_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/framework/formats/time_series_header.proto";
diff --git a/mediapipe/calculators/tensorflow/tensor_to_vector_float_calculator_options.proto b/mediapipe/calculators/tensorflow/tensor_to_vector_float_calculator_options.proto
index c9aa67f5..17dd18ee 100644
--- a/mediapipe/calculators/tensorflow/tensor_to_vector_float_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/tensor_to_vector_float_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensor_to_vector_int_calculator_options.proto b/mediapipe/calculators/tensorflow/tensor_to_vector_int_calculator_options.proto
index 76b9be95..ea047ae3 100644
--- a/mediapipe/calculators/tensorflow/tensor_to_vector_int_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/tensor_to_vector_int_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensor_to_vector_string_calculator_options.proto b/mediapipe/calculators/tensorflow/tensor_to_vector_string_calculator_options.proto
index 74df1be6..e1cfe283 100644
--- a/mediapipe/calculators/tensorflow/tensor_to_vector_string_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/tensor_to_vector_string_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensorflow_inference_calculator.proto b/mediapipe/calculators/tensorflow/tensorflow_inference_calculator.proto
index a243412c..ad1fb044 100644
--- a/mediapipe/calculators/tensorflow/tensorflow_inference_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensorflow_inference_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_calculator.proto b/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_calculator.proto
index 87b2304a..fe379810 100644
--- a/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "tensorflow/core/protobuf/config.proto";
diff --git a/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_generator.proto b/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_generator.proto
index 4643b4d6..461683aa 100644
--- a/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_generator.proto
+++ b/mediapipe/calculators/tensorflow/tensorflow_session_from_frozen_graph_generator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/packet_generator.proto";
 import "tensorflow/core/protobuf/config.proto";
diff --git a/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_calculator.proto b/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_calculator.proto
index 515b46fa..78bb11aa 100644
--- a/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_calculator.proto
+++ b/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "tensorflow/core/protobuf/config.proto";
diff --git a/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_generator.proto b/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_generator.proto
index d45fcb66..18f88c03 100644
--- a/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_generator.proto
+++ b/mediapipe/calculators/tensorflow/tensorflow_session_from_saved_model_generator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/packet_generator.proto";
 import "tensorflow/core/protobuf/config.proto";
diff --git a/mediapipe/calculators/tensorflow/unpack_media_sequence_calculator.proto b/mediapipe/calculators/tensorflow/unpack_media_sequence_calculator.proto
index 6c753d3b..c397247b 100644
--- a/mediapipe/calculators/tensorflow/unpack_media_sequence_calculator.proto
+++ b/mediapipe/calculators/tensorflow/unpack_media_sequence_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/calculators/core/packet_resampler_calculator.proto";
 import "mediapipe/framework/calculator.proto";
diff --git a/mediapipe/calculators/tensorflow/vector_float_to_tensor_calculator_options.proto b/mediapipe/calculators/tensorflow/vector_float_to_tensor_calculator_options.proto
index 01be3b72..ebb6278a 100644
--- a/mediapipe/calculators/tensorflow/vector_float_to_tensor_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/vector_float_to_tensor_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tensorflow/vector_int_to_tensor_calculator_options.proto b/mediapipe/calculators/tensorflow/vector_int_to_tensor_calculator_options.proto
index 65554bb1..6b8fd5a2 100644
--- a/mediapipe/calculators/tensorflow/vector_int_to_tensor_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/vector_int_to_tensor_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "tensorflow/core/framework/types.proto";
diff --git a/mediapipe/calculators/tensorflow/vector_string_to_tensor_calculator_options.proto b/mediapipe/calculators/tensorflow/vector_string_to_tensor_calculator_options.proto
index 908d98df..39bdd07c 100644
--- a/mediapipe/calculators/tensorflow/vector_string_to_tensor_calculator_options.proto
+++ b/mediapipe/calculators/tensorflow/vector_string_to_tensor_calculator_options.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/ssd_anchors_calculator.proto b/mediapipe/calculators/tflite/ssd_anchors_calculator.proto
index 3b1e3670..4bc1b322 100644
--- a/mediapipe/calculators/tflite/ssd_anchors_calculator.proto
+++ b/mediapipe/calculators/tflite/ssd_anchors_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_converter_calculator.proto b/mediapipe/calculators/tflite/tflite_converter_calculator.proto
index 5ed70879..4e907b38 100644
--- a/mediapipe/calculators/tflite/tflite_converter_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_converter_calculator.proto
@@ -1,6 +1,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_custom_op_resolver_calculator.proto b/mediapipe/calculators/tflite/tflite_custom_op_resolver_calculator.proto
index 546165a9..4b10791f 100644
--- a/mediapipe/calculators/tflite/tflite_custom_op_resolver_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_custom_op_resolver_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_inference_calculator.proto b/mediapipe/calculators/tflite/tflite_inference_calculator.proto
index 3b4d2896..8b7e774a 100644
--- a/mediapipe/calculators/tflite/tflite_inference_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_inference_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_tensors_to_classification_calculator.proto b/mediapipe/calculators/tflite/tflite_tensors_to_classification_calculator.proto
index c6c9d915..7786bd8a 100644
--- a/mediapipe/calculators/tflite/tflite_tensors_to_classification_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_tensors_to_classification_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_tensors_to_detections_calculator.proto b/mediapipe/calculators/tflite/tflite_tensors_to_detections_calculator.proto
index 41ad903d..731abe58 100644
--- a/mediapipe/calculators/tflite/tflite_tensors_to_detections_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_tensors_to_detections_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_tensors_to_landmarks_calculator.proto b/mediapipe/calculators/tflite/tflite_tensors_to_landmarks_calculator.proto
index ea390247..37bc6321 100644
--- a/mediapipe/calculators/tflite/tflite_tensors_to_landmarks_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_tensors_to_landmarks_calculator.proto
@@ -17,6 +17,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/tflite/tflite_tensors_to_segmentation_calculator.proto b/mediapipe/calculators/tflite/tflite_tensors_to_segmentation_calculator.proto
index fa768efe..47e76854 100644
--- a/mediapipe/calculators/tflite/tflite_tensors_to_segmentation_calculator.proto
+++ b/mediapipe/calculators/tflite/tflite_tensors_to_segmentation_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/annotation_overlay_calculator.proto b/mediapipe/calculators/util/annotation_overlay_calculator.proto
index 339bb218..70200799 100644
--- a/mediapipe/calculators/util/annotation_overlay_calculator.proto
+++ b/mediapipe/calculators/util/annotation_overlay_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/association_calculator.proto b/mediapipe/calculators/util/association_calculator.proto
index ca66f80b..873f29bf 100644
--- a/mediapipe/calculators/util/association_calculator.proto
+++ b/mediapipe/calculators/util/association_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/collection_has_min_size_calculator.proto b/mediapipe/calculators/util/collection_has_min_size_calculator.proto
index f482277c..135ceb2a 100644
--- a/mediapipe/calculators/util/collection_has_min_size_calculator.proto
+++ b/mediapipe/calculators/util/collection_has_min_size_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/detection_label_id_to_text_calculator.proto b/mediapipe/calculators/util/detection_label_id_to_text_calculator.proto
index eedc5e70..b7fefa9c 100644
--- a/mediapipe/calculators/util/detection_label_id_to_text_calculator.proto
+++ b/mediapipe/calculators/util/detection_label_id_to_text_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/label_map.proto";
diff --git a/mediapipe/calculators/util/detections_to_rects_calculator.proto b/mediapipe/calculators/util/detections_to_rects_calculator.proto
index d49eb6c5..b73b29c7 100644
--- a/mediapipe/calculators/util/detections_to_rects_calculator.proto
+++ b/mediapipe/calculators/util/detections_to_rects_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/detections_to_render_data_calculator.proto b/mediapipe/calculators/util/detections_to_render_data_calculator.proto
index acc847dc..b628651d 100644
--- a/mediapipe/calculators/util/detections_to_render_data_calculator.proto
+++ b/mediapipe/calculators/util/detections_to_render_data_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/filter_detections_calculator.proto b/mediapipe/calculators/util/filter_detections_calculator.proto
index 2b23236d..65ea5050 100644
--- a/mediapipe/calculators/util/filter_detections_calculator.proto
+++ b/mediapipe/calculators/util/filter_detections_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/labels_to_render_data_calculator.proto b/mediapipe/calculators/util/labels_to_render_data_calculator.proto
index 7946ff68..5b4e2664 100644
--- a/mediapipe/calculators/util/labels_to_render_data_calculator.proto
+++ b/mediapipe/calculators/util/labels_to_render_data_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/landmark_projection_calculator.proto b/mediapipe/calculators/util/landmark_projection_calculator.proto
index 221adbe0..b1bc637f 100644
--- a/mediapipe/calculators/util/landmark_projection_calculator.proto
+++ b/mediapipe/calculators/util/landmark_projection_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/landmarks_refinement_calculator.proto b/mediapipe/calculators/util/landmarks_refinement_calculator.proto
index e5234e71..ce697fe7 100644
--- a/mediapipe/calculators/util/landmarks_refinement_calculator.proto
+++ b/mediapipe/calculators/util/landmarks_refinement_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/landmarks_smoothing_calculator.proto b/mediapipe/calculators/util/landmarks_smoothing_calculator.proto
index 017facb3..1ebad90d 100644
--- a/mediapipe/calculators/util/landmarks_smoothing_calculator.proto
+++ b/mediapipe/calculators/util/landmarks_smoothing_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator_options.proto";
 
diff --git a/mediapipe/calculators/util/landmarks_to_detection_calculator.proto b/mediapipe/calculators/util/landmarks_to_detection_calculator.proto
index b5a56366..62e59bd2 100644
--- a/mediapipe/calculators/util/landmarks_to_detection_calculator.proto
+++ b/mediapipe/calculators/util/landmarks_to_detection_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/landmarks_to_floats_calculator.proto b/mediapipe/calculators/util/landmarks_to_floats_calculator.proto
index 310251e7..4d951237 100644
--- a/mediapipe/calculators/util/landmarks_to_floats_calculator.proto
+++ b/mediapipe/calculators/util/landmarks_to_floats_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/landmarks_to_render_data_calculator.proto b/mediapipe/calculators/util/landmarks_to_render_data_calculator.proto
index 99091954..24d4d92e 100644
--- a/mediapipe/calculators/util/landmarks_to_render_data_calculator.proto
+++ b/mediapipe/calculators/util/landmarks_to_render_data_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/latency.proto b/mediapipe/calculators/util/latency.proto
index b320d2c4..ef584f56 100644
--- a/mediapipe/calculators/util/latency.proto
+++ b/mediapipe/calculators/util/latency.proto
@@ -3,6 +3,7 @@ syntax = "proto2";
 
 // TODO: Switch to package mediapipe.
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 // Contains the latency information for a packet stream in mediapipe. The
 // following are provided
diff --git a/mediapipe/calculators/util/local_file_contents_calculator.proto b/mediapipe/calculators/util/local_file_contents_calculator.proto
index 17876c89..5cb88050 100644
--- a/mediapipe/calculators/util/local_file_contents_calculator.proto
+++ b/mediapipe/calculators/util/local_file_contents_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/logic_calculator.proto b/mediapipe/calculators/util/logic_calculator.proto
index fe00a2d9..e57d18d4 100644
--- a/mediapipe/calculators/util/logic_calculator.proto
+++ b/mediapipe/calculators/util/logic_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/non_max_suppression_calculator.proto b/mediapipe/calculators/util/non_max_suppression_calculator.proto
index 5fa96049..f4e4e2e8 100644
--- a/mediapipe/calculators/util/non_max_suppression_calculator.proto
+++ b/mediapipe/calculators/util/non_max_suppression_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/packet_frequency.proto b/mediapipe/calculators/util/packet_frequency.proto
index bb636a0a..121d3e2d 100644
--- a/mediapipe/calculators/util/packet_frequency.proto
+++ b/mediapipe/calculators/util/packet_frequency.proto
@@ -2,6 +2,7 @@ syntax = "proto2";
 
 // TODO: Switch to package mediapipe.
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 // Contains the packet frequency information.
 message PacketFrequency {
diff --git a/mediapipe/calculators/util/packet_frequency_calculator.proto b/mediapipe/calculators/util/packet_frequency_calculator.proto
index 396285d2..2266c5c1 100644
--- a/mediapipe/calculators/util/packet_frequency_calculator.proto
+++ b/mediapipe/calculators/util/packet_frequency_calculator.proto
@@ -16,6 +16,7 @@ syntax = "proto2";
 
 // TODO: Switch to package mediapipe.
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/packet_latency_calculator.proto b/mediapipe/calculators/util/packet_latency_calculator.proto
index 2350618e..60b93b20 100644
--- a/mediapipe/calculators/util/packet_latency_calculator.proto
+++ b/mediapipe/calculators/util/packet_latency_calculator.proto
@@ -16,6 +16,7 @@ syntax = "proto2";
 
 // TODO: Switch to package mediapipe.
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/rect_to_render_data_calculator.proto b/mediapipe/calculators/util/rect_to_render_data_calculator.proto
index 7611e3e0..fede02e3 100644
--- a/mediapipe/calculators/util/rect_to_render_data_calculator.proto
+++ b/mediapipe/calculators/util/rect_to_render_data_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/rect_to_render_scale_calculator.proto b/mediapipe/calculators/util/rect_to_render_scale_calculator.proto
index 377b1241..fbd0aa5f 100644
--- a/mediapipe/calculators/util/rect_to_render_scale_calculator.proto
+++ b/mediapipe/calculators/util/rect_to_render_scale_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/rect_transformation_calculator.proto b/mediapipe/calculators/util/rect_transformation_calculator.proto
index 44e781d4..3f8b77e4 100644
--- a/mediapipe/calculators/util/rect_transformation_calculator.proto
+++ b/mediapipe/calculators/util/rect_transformation_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/refine_landmarks_from_heatmap_calculator.proto b/mediapipe/calculators/util/refine_landmarks_from_heatmap_calculator.proto
index eebcaed0..70be0802 100644
--- a/mediapipe/calculators/util/refine_landmarks_from_heatmap_calculator.proto
+++ b/mediapipe/calculators/util/refine_landmarks_from_heatmap_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/thresholding_calculator.proto b/mediapipe/calculators/util/thresholding_calculator.proto
index b8d81ad5..f115af31 100644
--- a/mediapipe/calculators/util/thresholding_calculator.proto
+++ b/mediapipe/calculators/util/thresholding_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/timed_box_list_id_to_label_calculator.proto b/mediapipe/calculators/util/timed_box_list_id_to_label_calculator.proto
index 95cf7659..b44bf119 100644
--- a/mediapipe/calculators/util/timed_box_list_id_to_label_calculator.proto
+++ b/mediapipe/calculators/util/timed_box_list_id_to_label_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/timed_box_list_to_render_data_calculator.proto b/mediapipe/calculators/util/timed_box_list_to_render_data_calculator.proto
index 31e1ecbe..66859f29 100644
--- a/mediapipe/calculators/util/timed_box_list_to_render_data_calculator.proto
+++ b/mediapipe/calculators/util/timed_box_list_to_render_data_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/color.proto";
diff --git a/mediapipe/calculators/util/top_k_scores_calculator.proto b/mediapipe/calculators/util/top_k_scores_calculator.proto
index 08fb7a75..9c044c7b 100644
--- a/mediapipe/calculators/util/top_k_scores_calculator.proto
+++ b/mediapipe/calculators/util/top_k_scores_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/util/visibility_copy_calculator.proto b/mediapipe/calculators/util/visibility_copy_calculator.proto
index df25937b..82055ea1 100644
--- a/mediapipe/calculators/util/visibility_copy_calculator.proto
+++ b/mediapipe/calculators/util/visibility_copy_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator_options.proto";
 
diff --git a/mediapipe/calculators/util/visibility_smoothing_calculator.proto b/mediapipe/calculators/util/visibility_smoothing_calculator.proto
index 3b991923..ea621379 100644
--- a/mediapipe/calculators/util/visibility_smoothing_calculator.proto
+++ b/mediapipe/calculators/util/visibility_smoothing_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator_options.proto";
 
diff --git a/mediapipe/calculators/video/box_detector_calculator.proto b/mediapipe/calculators/video/box_detector_calculator.proto
index f0eb42f1..9fe3dd45 100644
--- a/mediapipe/calculators/video/box_detector_calculator.proto
+++ b/mediapipe/calculators/video/box_detector_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/tracking/box_detector.proto";
diff --git a/mediapipe/calculators/video/box_tracker_calculator.proto b/mediapipe/calculators/video/box_tracker_calculator.proto
index 20010820..6e0e43a4 100644
--- a/mediapipe/calculators/video/box_tracker_calculator.proto
+++ b/mediapipe/calculators/video/box_tracker_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/tracking/box_tracker.proto";
diff --git a/mediapipe/calculators/video/flow_packager_calculator.proto b/mediapipe/calculators/video/flow_packager_calculator.proto
index c1e91d8c..7a78b7b2 100644
--- a/mediapipe/calculators/video/flow_packager_calculator.proto
+++ b/mediapipe/calculators/video/flow_packager_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/tracking/flow_packager.proto";
diff --git a/mediapipe/calculators/video/flow_to_image_calculator.proto b/mediapipe/calculators/video/flow_to_image_calculator.proto
index 6d5fb845..9ba597b6 100644
--- a/mediapipe/calculators/video/flow_to_image_calculator.proto
+++ b/mediapipe/calculators/video/flow_to_image_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/video/motion_analysis_calculator.proto b/mediapipe/calculators/video/motion_analysis_calculator.proto
index a2af77d1..b3accc13 100644
--- a/mediapipe/calculators/video/motion_analysis_calculator.proto
+++ b/mediapipe/calculators/video/motion_analysis_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/tracking/motion_analysis.proto";
diff --git a/mediapipe/calculators/video/opencv_video_encoder_calculator.proto b/mediapipe/calculators/video/opencv_video_encoder_calculator.proto
index a3a7af3a..1feeae28 100644
--- a/mediapipe/calculators/video/opencv_video_encoder_calculator.proto
+++ b/mediapipe/calculators/video/opencv_video_encoder_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
diff --git a/mediapipe/calculators/video/tracked_detection_manager_calculator.proto b/mediapipe/calculators/video/tracked_detection_manager_calculator.proto
index 44624258..3546365c 100644
--- a/mediapipe/calculators/video/tracked_detection_manager_calculator.proto
+++ b/mediapipe/calculators/video/tracked_detection_manager_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 import "mediapipe/util/tracking/tracked_detection_manager_config.proto";
diff --git a/mediapipe/calculators/video/video_pre_stream_calculator.proto b/mediapipe/calculators/video/video_pre_stream_calculator.proto
index 8e87ce99..7d875e55 100644
--- a/mediapipe/calculators/video/video_pre_stream_calculator.proto
+++ b/mediapipe/calculators/video/video_pre_stream_calculator.proto
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package mediapipe;
+option csharp_namespace = "Mediapipe.Net.Framework.Protobuf";
 
 import "mediapipe/framework/calculator.proto";
 
